### PR TITLE
Address lint warnings #2

### PR DESCRIPTION
**Pull request by a-palchikov**
## Description

This is a continuation of lint warning fixes.

## Type of change

* Internal change (not necessarily a bug fix or a new feature)

## TODOs

- [x] Self-review the change
- [ ] Perform manual testing
- [ ] Address review feedback

### CODE REVIEW

File: lib/install/dispatcher/dispatcher.go [hunk 1]
```diff
@@ -99,12 +103,14 @@ type Status byte
...
+	StatusUnknown Status = iota
 	// StatusCompleted indicates a completed operation
-	StatusCompleted Status = iota
+	StatusCompleted
 	// StatusCompletedPending indicates a completed operation
 	// but with installer still active
-	StatusCompletedPending Status = iota
+	StatusCompletedPending
+	// StatusAborted indicates an aborted operation
+	StatusAborted
```
Comment (wadells) [inline lib/install/dispatcher/dispatcher.go:113]: This (and the related case) seems like new code?

File: lib/pack/localpack/packageserver.go [hunk 2]
```diff
@@ -383,8 +384,12 @@ func (p *PackageServer) UnpackedPath(loc loc.Locator) (string, error) {
 }
 
 func (p *PackageServer) Unpack(loc loc.Locator, targetDir string) error {
-	var err error
+	return p.UnpackWithOptions(loc, targetDir, nil)
+}
 
+// UnpackWithOptions unpacks the package specified with loc in targetDir
+// using opts for extraction
+func (p *PackageServer) UnpackWithOptions(loc loc.Locator, targetDir string, opts *dockerarchive.TarOptions) (err error) {
```
Comment (wadells) [inline lib/pack/localpack/packageserver.go:392]: What is this new function for? It doesn't seem used except by the old `Unpack` -- in which case why add a new public function?

**Review by wadells — APPROVED** [review]

File: lib/pack/localpack/packageserver.go (hunk 2 above)
Comment (wadells) [reply lib/pack/localpack/packageserver.go:392]: Looks like this change shouldbe included in https://github.com/gravitational/gravity/pull/2498 instead -- that PR has the rest of the context.

File: lib/install/dispatcher/dispatcher.go (hunk 1 above)
Comment (a-palchikov) [reply lib/install/dispatcher/dispatcher.go:113]: Yes, it is. I'll move this to another PR.

File: lib/pack/localpack/packageserver.go (hunk 2 above)
Comment (a-palchikov) [reply lib/pack/localpack/packageserver.go]: Removed.

**Comment by a-palchikov** [comment]
@wadells I addressed all your comments, PTAL.

**Review by wadells — APPROVED** [review]
Please take #2495 first, and enable the appropriate files/linters when merging this PR -- so that we're consistent with our enforcement.

**Pull request merged**